pkgs/applications/emulators/bsnes/bsnes-hd/default.nix

```diff
@@ -36,6 +36,14 @@ stdenv.mkDerivation {
     # be set to $out, so this will result in the .app ending up in the
     # Applications directory in the current nix profile.
     ./macos-copy-app-to-prefix.patch
+
+    # Fix build against gcc-13:
+    #   https://github.com/DerKoun/bsnes-hd/pull/124
```

Comment:
```suggestion
    #  https://github.com/DerKoun/bsnes-hd/pull/124
```

optional

Reply:
aligned to to the sentence right above, it is 2-space indented